crates/control/src/motion/stand_up_front.rs

```diff
@@ -35,6 +37,7 @@ pub struct CycleContext {
 #[derive(Default)]
 pub struct MainOutputs {
     pub stand_up_front_positions: MainOutput<Joints<f32>>,
+    pub stand_up_front_remaining_duration: MainOutput<Duration>,
```

Comment:
Same here as in stand_up_back of course :)

Reply:
Done!